pkg/agent/route/route_linux.go

```diff
@@ -1267,37 +1267,53 @@ func (c *Client) initNodeLatency() {
 	}
 
 	buildInputRule := func(ipProtocol iptables.Protocol, icmpType int32) string {
+		var comment string
+		if icmpType == 128 {
+			comment = "Antrea: allow ICMP echo request ingress packets from NodeLatencyMonitor"
+		} else if icmpType == 8 {
+			comment = "Antrea: allow ICMP echo ingress packets from NodeLatencyMonitor"
+		} else {
+			comment = "Antrea: allow ICMP echo reply ingress packets from NodeLatencyMonitor"
+		}
 		return iptables.NewRuleBuilder(antreaInputChain).
 			MatchInputInterface(gateway).
 			MatchICMP(&icmpType, nil, ipProtocol).
-			SetComment("Antrea: allow ICMP ingress packets from NodeLatencyMonitor").
+			SetComment(comment).
 			SetTarget(iptables.AcceptTarget).
 			Done().
 			GetRule()
 	}
 	buildOutputRule := func(ipProtocol iptables.Protocol, icmpType int32) string {
+		var comment string
+		if icmpType == 128 {
+			comment = "Antrea: allow ICMP echo request egress packets from NodeLatencyMonitor"
+		} else if icmpType == 8 {
+			comment = "Antrea: allow ICMP echo egress packets from NodeLatencyMonitor"
+		} else {
+			comment = "Antrea: allow ICMP echo reply egress packets from NodeLatencyMonitor"
+		}
 		return iptables.NewRuleBuilder(antreaOutputChain).
 			MatchOutputInterface(gateway).
 			MatchICMP(&icmpType, nil, ipProtocol).
-			SetComment("Antrea: allow ICMP egress packets from NodeLatencyMonitor").
+			SetComment(comment).
 			SetTarget(iptables.AcceptTarget).
 			Done().
 			GetRule()
 	}
 
 	if c.networkConfig.IPv6Enabled {
-		inputRequestRule := buildInputRule(iptables.ProtocolIPv6, int32(ipv6.ICMPTypeEchoRequest.Protocol()))
-		outputRequestRule := buildOutputRule(iptables.ProtocolIPv6, int32(ipv6.ICMPTypeEchoRequest.Protocol()))
-		inputReplyRule := buildInputRule(iptables.ProtocolIPv6, int32(ipv6.ICMPTypeEchoReply.Protocol()))
-		outputReplyRule := buildOutputRule(iptables.ProtocolIPv6, int32(ipv6.ICMPTypeEchoReply.Protocol()))
+		inputRequestRule := buildInputRule(iptables.ProtocolIPv6, int32(ipv6.ICMPTypeEchoRequest))
+		outputRequestRule := buildOutputRule(iptables.ProtocolIPv6, int32(ipv6.ICMPTypeEchoRequest))
+		inputReplyRule := buildInputRule(iptables.ProtocolIPv6, int32(ipv6.ICMPTypeEchoReply))
+		outputReplyRule := buildOutputRule(iptables.ProtocolIPv6, int32(ipv6.ICMPTypeEchoReply))
 		c.nodeLatencyMonitorIPTablesIPv6.Store(antreaInputChain, []string{inputRequestRule, inputReplyRule})
 		c.nodeLatencyMonitorIPTablesIPv6.Store(antreaOutputChain, []string{outputRequestRule, outputReplyRule})
```

Comment:
I would avoid fining these extra variables, for readability, I think they are not necessary:
```go
c.nodeLatencyMonitorIPTablesIPv6.Store(antreaInputChain, []string{
        buildInputRule(iptables.ProtocolIPv6, int32(ipv6.ICMPTypeEchoRequest)),
        buildInputRule(iptables.ProtocolIPv6, int32(ipv6.ICMPTypeEchoReply)),
})
```